enterprise/metrics/src/main/java/org/neo4j/metrics/source/ClusterMetrics.java

```diff
@@ -56,6 +72,7 @@ public void start() throws Throwable
     {
         if ( config.get( MetricsSettings.neoClusterEnabled ) )
```

Comment:
this will actually make the old code better, since we'll register the `SLAVE_PULL_UPDATES` metrics only if we are in cluster mode.